docs/topics/cli-run.adoc

```diff
@@ -63,13 +65,27 @@ The following command analyzes the `com.acme` and `org.apache` packages of the l
 
 [source,terminal,subs="attributes+"]
 ----
-$ <{ProductShortName}_HOME>/bin/{mta-cli} --input /path/to/jee-example-app-1.0.0.ear \
+$ <{ProductShortName}_HOME>/bin/{mta-cli} analyze --input /path/to/jee-example-app-1.0.0.ear \
     --output /path/to/report-output/ \
     --target eap:7 --target cloud-readiness --packages com.acme org.apache
 ----
 []
 
 [discrete]
+<<<<<<< HEAD
+=======
+=== Overriding {ProductShortName} properties
+
+To override the default _Fernflower_ decompiler, pass the `-Dwindup.decompiler` argument on the command line. For example, to use the _Procyon_ decompiler, use the following syntax:
+
+[source,terminal,subs="attributes+"]
+----
+$ <{ProductShortName}_HOME>/bin/{mta-cli} -Dwindup.decompiler=procyon \
+    --input <INPUT_ARCHIVE_OR_DIRECTORY> --output <OUTPUT_REPORT_DIRECTORY> \
+    --target <TARGET_TECHNOLOGY> --packages <PACKAGE_1> <PACKAGE_2>
+----
+
```

Comment:
This section can be removed as the new mta-cli version doesn't allow to override the default decompiler

Reply:
Removed in PR - https://github.com/windup/windup-documentation/pull/828